.circleci/config.yml

```diff
@@ -106,6 +106,10 @@ jobs:
           command: |
             source env/bin/activate
             pip install --requirement requirements.txt --requirement requirements/test.txt
+      - run:
+          name: Check dependencies compatibility
```

Comment:
I feel there is sufficient clarity in the `check` output to avoid creating a separate job step. What do you think?

If we vote to keep it, I feel “compatibility” does not reveal that we want to discover broken dependency trees (conflicting versions required with no resolution possible).

Reply:
I don't mind. I slightly prefer the separate job step for clarity. But we should consistently combine or separate across all five repos.

So, consolidate job steps or rename to "Check for conflicting dependencies"?